src/components/Modeler.vue

```diff
@@ -602,22 +610,16 @@ export default {
       return new Promise(resolve => setTimeout(resolve, cursorWaitTime));
     },
     async renderPaper() {
-      await this.$nextTick();
+      this.isRendering = true;
+
       await this.paperManager.performAtomicAction(async() => {
-        this.isRendering = true;
         await this.waitForCursorToChange();
-        this.paperManager.awaitScheduledUpdates().then(() => this.isRendering = false);
```

Comment:
Since we're in an `async` function, and to keep things consistent, I switched to use `await`. This made it easier for me to read.